mssql-plugin/src/main/java/io/cdap/plugin/mssql/SqlServerSourceDBRecord.java

```diff
@@ -75,14 +110,22 @@ protected void handleField(ResultSet resultSet, StructuredRecord.Builder recordB
         recordBuilder.setTime(field.getName(),
             timestamp == null ? null : timestamp.toLocalDateTime().toLocalTime());
         break;
-      case SqlServerSourceSchemaReader.DATETIME_OFFSET_TYPE:
-        recordBuilder.set(field.getName(), resultSet.getString(columnIndex));
-        break;
       default:
         setField(resultSet, recordBuilder, field, columnIndex, sqlType, sqlPrecision, sqlScale);
     }
   }
 
+  private void setZonedDateTimeBasedOnOuputSchema(StructuredRecord.Builder recordBuilder,
```

Comment:
We can move this method's content into the caller block and update the comment about which flow is deprecated.

Reply:
done